rhobs/alerting/data_plane/prometheus.gitops_cluster_agent_op_alerts.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-cluster-agent-operations-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: cluster-agent-operations_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsClusterAgentOperationErrors
+      expr: |
+        sum(operationDB_rows_in_non_complete_state) <= 10
+      for: 5m
+      labels:
+        severity: warning
+        slo: true
+      annotations:
+        summary: >-
+          Cluster-Agent Operations in non-complete state are too high
```

Comment:
I suggest including `{{ $value }}` in the summary/description to have better visibility of the query result.